packages/xo-web/src/common/xo/index.js

```diff
@@ -1893,6 +1893,13 @@ export const exportVdi = async vdi => {
   })
 
   info(_('startVdiExport'), vdi.id)
+
+  if (format === 'raw') {
+    return fetch(`./rest/v0/vdis/${resolveId(vdi)}.raw`).then(res => {
+      window.open(res.url)
+    })
```

Comment:
The REST API does not return an URL, simply open the REST API URL in a new window, it will work directly.